02/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const links = document.querySelectorAll("a[data-url]");
+if (links) {
```

Comment:
Tutaj `if` jest nie potrzebny bo `.querySelectorAll` zwraca zawszę "prawdę" - jest to tablica, nawet usta tablica jest  "prawdą". Jak już to `if(links.length > 0)`, ale pętla i tak się uruchomi jak jest coś w tablicy więc ten `if` w tym przypadku nic nie daje.